tests/test_routes.py

```diff
@@ -71,3 +72,30 @@ def test_search(client):
         ).json
 
         assert len(response_json) == len(rows)
+
+
+@pytest.mark.parametrize("sub_criteria_response_keys", ["id", "name", "score_submitted","themes"])
+def test_get_sub_criteria(request, client, sub_criteria_response_keys):
+    """Test to check that sub criteria metadata and ordered themes are returned for
+    a COFR2W2 sub criteria"""
+
+    sub_criteria_id = "benefits"
+    response_json = client.get(
+        f"/sub_criteria_overview/{sub_criteria_id}"
+    ).json
+
+    assert sub_criteria_response_keys in response_json 
```

Comment:
If we care about the order here, should we assert on the keys directly as an array rather than doing a substring match in the JSON?

Reply:
Yes you're right! I've added another check to ensure we are ordering the themes as expected.